ddtrace/profiling/exporter/http.py

```diff
@@ -97,23 +107,20 @@ def _encode_multipart_formdata(fields, tags):
     def _get_tags(self, service):
         tags = {
             "service": service.encode("utf-8"),
-            "host": HOSTNAME.encode("utf-8"),
             "runtime-id": runtime.get_runtime_id().encode("ascii"),
-            "language": b"python",
-            "runtime": PYTHON_IMPLEMENTATION,
-            "runtime_version": PYTHON_VERSION,
-            "profiler_version": ddtrace.__version__.encode("utf-8"),
         }
 
+        user_tags = parse_tags_str(os.environ.get("DD_TAGS", {}))
```

Comment:
should we parse these env vars once at creation rather than on every export?

Reply:
We could, that'd break changing them at runtime. Not sure it's a big deal?

Reply:
Not sure I follow what you mean by changing at runtime?

If the user via code updates the tags or if they modify env variables after the process has started?

Reply:
If they modify the env variables after the process has started.